### PR TITLE
Docs: added page for running a node with latitude

### PR DESCRIPTION
**Pull request by meaghanfitzgerald**
Added a page for running a node with [Latitude.sh](https://github.com/NottherealIllest/Latitude.sh-post/blob/main/avalanhe/avax-copy.md) to the build menu where the node tutorials currently exist

### CODE REVIEW

**Comment by meaghanfitzgerald** [comment]
I currently am not able to use markdownlint to clean up my files, even though I have:

1.installed markdownlint on vscode and set it as my default formatter, then called `Opt+Shift+F`
2.`Opt+shift+P` and chosen "Format Document"
3. Hovered over the error, selected "Quick Fix", then "Fix all supported markdownlint violations in this document"
4. installed the CLI, set my config file to this repo's `.markdownlint.json` and invoked it there

see video for demo of markdown doing nothing: 
https://user-images.githubusercontent.com/99047250/234176788-e23e5b9b-ac77-4661-832c-5ac56db6b28d.mov

File: docs/nodes/build/set-up-an-avalanche-node-with-latitude.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+:::
+
+After gaining access, we’ll need to set up our Avalanche node. To do this, follow the instructions
+ here to install and run your node [Set Up Avalanche Node With Installer](https://docs.avax.network/nodes/build/set-up-node-with-installer).
+
+Your AvalancheGo node should now be running and in the process of bootstrapping, which can take a few
+ hours. To check if it's done, you can issue an API call using `curl`.
+The request is:
+
+<<<<<<< HEAD
```
Comment (yulin-dong) [inline docs/nodes/build/set-up-an-avalanche-node-with-latitude.md:62]: There are quite a few left-over by git merge which should be cleaned up.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/nodes/build/set-up-an-avalanche-node-with-latitude.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    "method" :"info.isBootstrapped",
+    "params": {
+        "chain":"X"
+    }
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/info
+```
+
+Once the node is finished bootstrapping, the response will be:
+
+```json
```
Comment (yulin-dong) [inline docs/nodes/build/set-up-an-avalanche-node-with-latitude.md:64]: Please format this json text.

File: docs/nodes/build/set-up-an-avalanche-node-with-latitude.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+curl -X POST --data '{
+"jsonrpc":"2.0",
+"id" :1,
+"method" :"info.getNodeID"
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/info
+```
+
+The response contains the node ID.
+
+`{"jsonrpc":"2.0","result":{"nodeID":"KhDnAoZDW8iRJ3F26iQgK5xXVFMPcaYeu"},"id":1}`
```
Comment (yulin-dong) [inline docs/nodes/build/set-up-an-avalanche-node-with-latitude.md:87]: json format

File: docs/nodes/build/set-up-an-avalanche-node-with-latitude.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+},
+"id": 1
+}
+```
+
+You can continue on, even if AvalancheGo isn't done bootstrapping.
+In order to make your node a validator, you'll need its node ID. To get it, run:
+
+```sh
+curl -X POST --data '{
```
Comment (yulin-dong) [inline docs/nodes/build/set-up-an-avalanche-node-with-latitude.md:78]: json format

**Comment by yulin-dong** [comment]
LGTM for the format. Please get an approval for the content.

**Review by dubrado — APPROVED** [review]

**Review by amandamarquis1 — APPROVED** [review]
Approving changes

**Review by dubrado — APPROVED** [review]

**Review by dubrado — APPROVED** [review]

**Review by dubrado — APPROVED** [review]

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**